src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -125,7 +125,7 @@ public double RenderFrequency
         /// <summary>
         /// Gets a double representing the time spent in the RenderFrame function, in seconds.
         /// </summary>
-        [Obsolete("Use UpdateTime instead. We no longer separate UpdateFrame and RenderFrame.", true)]
+        [Obsolete("Use UpdateFrequency instead. We no longer separate UpdateFrame and RenderFrame.", true)]
```

Comment:
This one was correct before, `UpdateTime` is the property that should be used instead of `RenderTime`. 
But I think I've broken `UpdateTime`, I've opened an issue for that as of #1641

Reply:
So, if this PR serves it's purpose then please merge it, 
if not then, please suggest what more changes will be required ?